db/import.go

```diff
@@ -235,6 +235,7 @@ func (db *Database) importDoc(docid string, body Body, isDelete bool, existingDo
 		docOut = alreadyImportedDoc
 	case nil:
 		db.DbStats.SharedBucketImport().Add(base.StatKeyImportCount, 1)
+		db.DbStats.SharedBucketImport().Set(base.StatKeyImportHighSeq, base.ExpvarUInt64Val(docOut.syncData.Sequence))
```

Comment:
Switch to int64(docOut.syncData.Sequence) for reduced overhead, and probably improved visibility in the event we hit a bug that caused sequence to overflow int64.